Classes/BITCrashCXXExceptionHandler.mm

```diff
@@ -58,7 +58,7 @@
 
 @implementation BITCrashUncaughtCXXExceptionHandlerManager
 
-extern "C" void LIBCXXABI_NORETURN __cxa_throw(void *exception_object, std::type_info *tinfo, void (*dest)(void *))
```

Comment:
@diederich Thats so much better, I wonder why I didn't find that. I implemented this is #425